opencti-platform/opencti-front/tests_e2e/model/login.pageModel.ts

```diff
@@ -27,6 +27,9 @@ export class LoginPage {
     
     async login() {
         await this.page.goto('/');
+        await this.page.waitForTimeout(5000);
```

Comment:
Can you explain that?

Reply:
it's for debugging. I got some issue on the login page. I am waiting 5sec to be sure to load the page then try a screenshot. Will be removed after.